apps/api/src/users/schemas/create-user.schema.ts

```diff
@@ -0,0 +1,14 @@
+import { type Static, Type } from "@sinclair/typebox";
+
+export const createUserSchema = Type.Object({
+  email: Type.String({ format: "email" }),
+  firstName: Type.String({ minLength: 1, maxLength: 64 }),
+  lastName: Type.String({ minLength: 1, maxLength: 64 }),
+  role: Type.Union([
+    Type.Literal("admin"),
+    Type.Literal("student"),
```

Comment:
I think the administrator should be able to add other admins or tutors, and students will register themselves.